internal/sim/api.go

```diff
@@ -313,50 +315,81 @@ func validateWorkload(t reflect.Type) error {
 
 // Run runs simulations for the provided duration.
 func (s *Simulator) Run(duration time.Duration) Results {
+	// TODO(mwhittaker): Use a smarter algorithm to sweep over hyperparameters.
+	// TODO(mwhittaker): Read and run graveyard entries.
 	start := time.Now()
 	deadline := start.Add(duration)
 	ctx, cancel := context.WithDeadline(context.Background(), deadline)
 	defer cancel()
 
-	// TODO(mwhittaker): Use a smarter algorithm to sweep over hyperparameters.
-	// TODO(mwhittaker): Run simulations in multiple goroutines.
-	// TODO(mwhittaker): Read and run graveyard entries.
+	// Spawn n concurrent simulators. The simulators read options from the opts
+	// channel and write errors and results to the errs and results channels.
+	//
+	// TODO(mwhittaker): Optimize things and pick a smarter value of n.
+	n := 1000
+	opts := make(chan options, n)
+	errs := make(chan error, n)
+	results := make(chan Results, n)
```

Comment:
Rename errs and results to describe what type of errors they store.

Reply:
Done. I also added some more comments.